third_party/CMakeLists.txt

```diff
@@ -318,3 +385,17 @@ if (${AMBER_USE_LOCAL_VULKAN})
   set(CMAKE_CXX_FLAGS ${CXX_BACK})
 endif()
 
+if (${AMBER_ENABLE_SWIFTSHADER})
+  set(CXX_BACK ${CMAKE_CXX_FLAGS})
+  SET(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "${SWIFTSHADER_BUILD_FIXES}")
+  STRING(REGEX REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+  set(BUILD_EGL FALSE CACHE BOOL FALSE)
+  set(BUILD_GLESv2 FALSE CACHE BOOL FALSE)
+  set(BUILD_GLES_CM FALSE CACHE BOOL FALSE)
+  set(BUILD_VULKAN TRUE CACHE BOOL TRUE)
```

Comment:
Strange we don't need a docstring ?  Or, the docstrings here are FALSE and TRUE.... :-)

Should these be...
set(BUILD_EGL FALSE CACHE BOOL "Should SwiftShader build EGL?")
...

and the one that turns Vulkan on should also have FORCE:

set(BUILD_VULKAN FALSE CACHE BOOL "Should SwiftShader build Vulkan?" FORCE)

Reply:
Doing them as "" would match the rest of the file. We don't really expect these to be set, we just have to set them in to CACHE so cmake doesn't lose them.